### PR TITLE
COLLADA load morph target normals 

### PR DESCRIPTION
**Pull request by cheece**
if the file has normals use them!

### CODE REVIEW

File: tools/editor/io_plugins/editor_import_collada.cpp [hunk 1]
```diff
@@ -1354,8 +1354,60 @@ Error ColladaImport::_create_mesh_surfaces(Ref<Mesh>& p_mesh,const Map<String,Co
...
+					
+						//read normals from morph target
+						DVector<Vector3>::Write vertw = normals.write();
+
+						for(int m_i=0;m_i<m->array.size()/stride;m_i++) {
+
+							int pos = m_i*stride;
+							Vector3 vtx( m->array[pos+0], m->array[pos+1], m->array[pos+2] );
+
+	#ifndef NO_UP_AXIS_SWAP
```
Comment (gnat) [inline tools/editor/io_plugins/editor_import_collada.cpp]: Wrong indentation level.

File: tools/editor/io_plugins/editor_import_collada.cpp [hunk 2]
```diff
@@ -1354,8 +1354,60 @@ Error ColladaImport::_create_mesh_surfaces(Ref<Mesh>& p_mesh,const Map<String,Co
...
+							Collada::Vertex vertex;
+							vertex.vertex=vtx;
+							vertex.fix_unit_scale(collada);
+							vtx=vertex.vertex;
+
+							vtx = p_local_xform.xform(vtx);
+
+
+							if (vertex_map.has(m_i)) { //vertex may no longer be here, don't bother converting
+
```
Comment (gnat) [inline tools/editor/io_plugins/editor_import_collada.cpp]: Extra new lines need cleanup in general.

**Comment by jotson** [comment]
Getting a compile error on linux at the `std::cout` statements:

```
tools/editor/io_plugins/editor_import_collada.cpp:1363:7: error: 'cout' is not a member of 'std'
       std::cout << "has normals" << std::endl;
       ^
tools/editor/io_plugins/editor_import_collada.cpp:1363:37: error: 'endl' is not a member of 'std'
       std::cout << "has normals" << std::endl;
```

I think `<iostream>` needs to be included somewhere.

**Comment by UsernameIsAReservedWord** [comment]
same compilation error with MSVC2010 :

````
`tools\editor\io_plugins\editor_import_collada.cpp(1363) : error C2039: 'cout' : is not member of 'std'
tools\editor\io_plugins\editor_import_collada.cpp(1363) : error C2065: 'cout' : identifier not declared 
tools\editor\io_plugins\editor_import_collada.cpp(1363) : error C2039: 'endl' : n'est pas membre de 'std'
tools\editor\io_plugins\editor_import_collada.cpp(1363) : error C2065: 'endl' : identificateur non déclaré
tools\editor\io_plugins\editor_import_collada.cpp(1365) : error C2039: 'cout' : n'est pas membre de 'std'
tools\editor\io_plugins\editor_import_collada.cpp(1365) : error C2065: 'cout' : identificateur non déclaré
tools\editor\io_plugins\editor_import_collada.cpp(1365) : error C2039: 'endl' : n'est pas membre de 'std'
tools\editor\io_plugins\editor_import_collada.cpp(1365) : error C2065: 'endl' : identificateur non déclaré
scons: *** [tools\editor\io_plugins\editor_import_collada.windows.tools.32.obj] Error 2
scons: building terminated because of errors.
````

**Comment by UsernameIsAReservedWord** [comment]
also, just as a side-note, for verbose purpose, instead of using <iostream>, we could follow godot way of verbosing as it is shown on line 1157 for instance.

````
if (OS::get_singleton()->is_stdout_verbose())
    print_line("Collada: bla bla bla.");
````

**Pull request merged**